tests/e2e/e2e_setup_test.go

```diff
@@ -172,6 +172,7 @@ func (s *IntegrationTestSuite) SetupSuite() {
 	s.initValidatorConfigs(s.chainB)
 	s.runValidators(s.chainB, 10)
 
+	time.Sleep(10 * time.Second)
```

Comment:
Do we want some kind of facility where we retry the request with increasing backoff? Would be nicer & more efficient than playing with the time outs & sleeping. If this is interesting, then it's a new testing feature.